README.md

```diff
@@ -1,130 +1,209 @@
 
-### Import `sdk.js`:
 ```html
-<script src="https://cdn.smartcar.com/javascript-sdk/sdk-1.0.0.js"></script>
+<script src="https://javascript-sdk.smartcar.com/sdk-2.0.0.js"></script>
 ```
 
-### Use `sdk.js`:
+### Use `sdk.js`
+
 Initialize:
+
 ```javascript
 const smartcar = new Smartcar({
   clientId: `your-client-id`,
   redirectUri: `your-redirect-uri`,
-  scope: ['read_vehicle_info', 'read_odometer'], // optional
-  onComplete: function() { // optional
-    window.location.reload();
+  // optional, defaults to all scopes
+  scope: ['read_vehicle_info', 'read_odometer'],
+  // if using smartcar-hosting this is required and must take at least two
+  // arguments - error & code
+  // otherwise optional (will be called with the same arguments but is not
+  // required to handle them)
+  onComplete: function(error, code, [state]) {
+    // actions to take on completion of auth flow
+    // if using smartcar-hosting this should send the code to your back end
+    // server to exchange for an access token
   },
-  development: false, // optional
+  development: false, // optional, defaults to false
+  useSmartcarHostedRedirect: true, // optional, defaults to false
 });
 ```
-The best placement for the above code is just before the closing `</body>` tag.
 
-To open the Smartcar OAuth dialog directly:
-```javascript
-// note: state and forcePrompt are optional
-smartcar.openDialog({state, forcePrompt});
-```
+Here's an example `onComplete` function:
 
-To add a click handler to an HTML element that will open the Smartcar OAuth dialog:
 ```javascript
-// note: state and forcePrompt are optional
-smartcar.addClickHandler({id, state, forcePrompt});
+function(error, code
+if (error) {
+  // redirect user to error page
+} else {
+  // using the axios library
+  // posting to a domain specified by SERVER variable
+  axios
+    .post(`${SERVER}/auth`, {code})
+    .then((_) => {
+      // take action now that authorization is complete
+      // maybe pull odometer or location and display to your user?
+      // at this point the limit is your own creativity :)
+    });
+}
 ```
 
-You may embed `callback.js` in the page that is served by your redirectUri. Upon loading, this file will invoke the `onComplete` callback specified above and close the window:
-```html
-<script src="https://cdn.smartcar.com/javascript-sdk/callback-1.0.0.js"></script>
-```
+Once initialized there are three instance methods for setting up the OAuth flow:
 
-To retrieve the Smartcar OAuth URL (useful in single page applications when redirecting to Smartcar's OAuth flow in a new tab):
-```javascript
-// note: state and forcePrompt are optional
-smartcar.generateLink({state, forcePrompt});
-```
+- `openDialog()` - to open the Smartcar OAuth dialog directly
+- `addClickHandler({id})` - to add a click handler to an HTML element that will open the Smartcar OAuth dialog
```

Comment:
nit: `addClickHandler('element-id')` the `{}` make it seem like an options argument